app/components/record_details/view.rb

```diff
@@ -93,7 +93,9 @@ def record_created_row
     end
 
     def trainee_start_date_row
-      mappable_field(trainee_start_date, t(".trainee_start_date"), edit_trainee_start_date_path(trainee))
+      change_link = trainee.starts_course_in_the_future? ? nil : edit_trainee_start_date_path(trainee)
+
+      mappable_field(trainee_start_date, t(".trainee_start_date"), change_link)
```

Comment:
Might need to be agreed with @avinhurry and this PR - https://github.com/DFE-Digital/register-trainee-teachers/pull/1724/files

I _think_ you're both achieving the same thing!

Reply:
filthy overlap-ses